server/metrix/metric.py

```diff
@@ -80,6 +80,18 @@ def derivative_wrt_time(function, values, timestamps):
             actual_values = next_values
         return derivatives
 
+    @staticmethod
+    def calculate_distance(function, values):
```

Comment:
preco je tato funkcia tu a nie priamo v tej triede tej metriky?

Reply:
neviem, aj ostatné metriky tu majú funkcie, kedy ju nechat v triede tej metriky a kedy ju dat sem?

Reply:
Neviem, moj nazor je, ze pokial ze pokial nejaku funkciu pouzivaju viacere metriky, tak mat ju v nad-triede, nech nemusime mat duplikovany kod. Ale ak ta funkcia nema zmysel pre ine metriky tak nevidim dovod aby bola v nadtriede. Ak nejaky dovod tam vidis tak ju tam kludne nechaj.

Reply:
Ešte bude jedna metrika ,ktorú pridám v stredu a ta tiež použije túto funkciu